Libraries/Lists/FlatList.js

```diff
@@ -629,16 +655,54 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
     }
   };
 
+  _listItemComponent = (info: RenderItemProps<ItemT>) => {
+    const {ListItemComponent, numColumns, columnWrapperStyle} = this.props;
+
+    if (!ListItemComponent) {
+      return null;
+    }
+
+    if (numColumns > 1) {
```

Comment:
This is mostly copy-pasta - can you factor into a shared function?

Reply:
@sahrens My original approach was to use one function. I opted to use two functions as it was easier to read and if the renderItem prop was ever removed it'd be easy to remove _renderItem from FlatList. I'm open rolling back to a single function. Having two functions with very similar functionality is bound to get out of sync down the road.

Reply:
@sahrens I moved back to a single function